test/test_transforms.py

```diff
@@ -299,13 +299,18 @@ def test_pad(self):
         width = random.randint(10, 32) * 2
         img = torch.ones(3, height, width)
         padding = random.randint(1, 20)
+        fill = random.randint(1, 50)
         result = transforms.Compose([
             transforms.ToPILImage(),
-            transforms.Pad(padding),
+            transforms.Pad(padding, fill=fill),
             transforms.ToTensor(),
         ])(img)
         self.assertEqual(result.size(1), height + 2 * padding)
         self.assertEqual(result.size(2), width + 2 * padding)
+        # check that all elements in the padded region correspond
+        # to the pad value
+        self.assertTrue((result[:, :padding, :] == (fill / 255)).all())
+        self.assertTrue((result[:, :, :padding] == (fill / 255)).all())
```

Comment:
Are you sure you want to check for equality of `float`s? Might be flaky.

Reply:
I've made this an approximate check